ui/components/multichain/pages/send/components/recipient-input.tsx

```diff
@@ -75,15 +75,15 @@ export const SendPageRecipientInput = () => {
         }}
         onReset={() => dispatch(resetRecipientInput())}
         scanQrCode={() => {
```

Comment:
```suggestion
        scanQrCode={
          undefined
           // trackEvent({
          //   event: 'Used QR scanner',
          //   category: MetaMetricsEventCategory.Transactions,
          //   properties: {
          //     action: 'Edit Screen',
          //     legacy_event: true,
          //   },
          // });
          // dispatch(showQrScanner());
        }
```

Reply:
Could change this to undefined, and handle hiding the QR code button [here](https://github.com/MetaMask/metamask-extension/blob/e4672a884c72e3c4405ef5e81112236d4862ccd0/ui/pages/confirmations/send/send-content/add-recipient/domain-input.component.js#L175) with this code 


```
              {!this.props.onReset && !this.props.scanQrCode && (
                <ButtonIcon
                  className="ens-input__wrapper__action-icon-button"
                  onClick={() => {
                    if (userInput?.length > 0) {
                      this.props.onReset();
                    } else {
                      this.props.scanQrCode();
                    }
                  }}
                  iconName={userInput ? IconName.Close : IconName.Scan}
                  ariaLabel={t(userInput ? 'close' : 'scanQrCode')}
                  color={
                    userInput ? IconColor.iconDefault : IconColor.primaryDefault
                  }
                  data-testid="ens-qr-scan-button"
                />
              )}
```

Understand if we don't want to do this in case it blows up the scope of the PR, but can maybe include that in a fast follow for 12.7 so we don't leave a dead link in the UI